docker/Dockerfile.cpu

```diff
@@ -0,0 +1,19 @@
+FROM python:3.11
+
+RUN apt-get update && apt-get install -y \
+	ffmpeg
+
+WORKDIR /app
+
+ADD requirements.txt .
+RUN pip install -r requirements.txt
+
+ADD inswapper_128.onnx .
```

Comment:
All this ADD have to be removed - in next branch loading the models also happens automatically now

Reply:
It should be placed into cache then, so that it won't be downloaded on each run.